.github/components.yml

```diff
@@ -120,6 +120,12 @@ JS_API:
   revalidate:
     - LLM_samples
 
+CPP_API: # does not participate in smart CI
+  revalidate: []
+
+Python_API: # does not participate in smart CI
+  revalidate: []
```

Comment:
My understanding is that `components.yml` is for smart CI only. What's the point listing things that aren't in smart CI?

Reply:
it's turned out that smart CI uses all labels, I've excluded CPP and Python API in Smart CI action run.